pubsub/include/marlin/pubsub/PubSubNode.hpp

```diff
@@ -598,6 +602,13 @@ void PUBSUBNODETYPE::send_RESPONSE(
 	transport.send(std::move(m));
 }
 
+
+bool check_reward_worthy(uint8_t *buf, size_t capacity) {
```

Comment:
Avoid free functions like this in the header file, causes duplicate symbol issues with the linker. Also, this is going to be chain specific, so move this inside the ABCI as well.

Reply:
This function should probably also take a WeakBuffer instead of raw pointers.